docs/source/coder/globalKeys.rst

```diff
@@ -0,0 +1,281 @@
+
+All elements of a global event can be accessed directly.
+::
+
+    >>> index = ('b', ['ctrl', 'alt'])
+    >>> event.globalKeys[index].func
+    <function myfunc2 at 0x112eecb90>
+    >>> event.globalKeys[index].func_args
+    (1, 2)
+    >>> event.globalKeys[index].func_kwargs
+    {'foo': 3, 'bar': 4}
+    >>> event.globalKeys[index].name
+    'my name'
+
+Removing global event keys
+--------------------------
+There are three ways to remove global event keys:
+
+- using :func:`psychopy.event.globalKeys.remove`,
+- using `del`, and
+- using :func:`psychopy.event.globalKeys.clear`.
+
+:func:`psychopy.event.globalKeys.remove`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To remove a single key, pass the key name and modifiers (if any) to
+:func:`psychopy.event.globalKeys.remove`.
+::
+
+    >>> event.globalKeys.remove(key='a')
+
+`del`
+~~~~~
+Like with other dictionaries, items can be removed from `event.globalKeys`
+by using the `del` statement. The provided index key must be specified as
+described in :ref:`indexing`.
+::
+
+    >>> index = ('b', ['ctrl', 'alt'])
+    >>> del event.globalKeys[index]
+
+:func:`psychopy.event.globalKeys.clear`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+A convenience method to quickly delete *all* global event keys is
+:func:`psychopy.event.globalKeys.clear`.
+::
+
+    >>> event.globalKeys.clear()
+
+Global shutdown key
+-------------------
+
+The PsychoPy preferences for `shutdownKey` and `shutdownKeyModifiers`
+(both unset by default) will be used to automatically create a global
+shutdown key.
+::
+
+    >>> from psychopy.preferences import prefs
+    >>> prefs.general['shutdownKey'] = 'q'
+    >>> from psychopy import event
+    >>> event.globalKeys
+    <_GlobalEventKeys :
+        [Q] -> 'shutdown (auto-created from prefs)' <function quit at 0x10c171938>
+    >
+
+When importing the `psychopy.event` module, `event.globalKeys` was automatically
+initialized and a global shutdown key added, based on the preferences setting.
+This key is associated with the :func:~`psychopy.core.quit` function, which will
+shut down PsychoPy.
+::
+
+   >>> from psychopy.core import quit
+   >>> event.globalKeys['q'].func == quit
+   True
+
+A working example
+-----------------
+
+In the above code snippets, our global event keys were not actually functional,
+as we didn't create a window, which is required to actually collect the key
+presses. Our working example will thus first create a window and then add
+global event keys to change the window color and quit the experiment,
+respectively.
+::
+
+    #!/usr/bin/env python2
+    # -*- coding: utf-8 -*-
+
+    from __future__ import print_function
+    from psychopy import core, event, visual
+
+
+    def change_color(win, log=False):
+        if win.color == 'gray':
+            new_color = 'blue'
+        else:
+            new_color = 'gray'
+
+        win.color = new_color
+
+        if log:
+            print('Changed color to %s' % new_color)
+
+
+    def main():
```

Comment:
I did consider this, but the entire document somehow ended up being pretty technical anyway -- it's not really beginner-friendly. So I thought, if the target audience aren't total beginners anyway, might as well demonstrate "best practices" here, using functions and avoiding global variables... Really not sure about this!